### PR TITLE
Proofread documentation

### PR DESCRIPTION
**Pull request by Mogztter**
## Kind of change

- [ ] Bug fix
- [ ] New non-breaking feature
- [ ] New breaking feature
- [x] Documentation update
- [ ] Build improvement

### CODE REVIEW

**Review by robertpanzer — APPROVED** [review]

File: docs/modules/ROOT/pages/conversion-examples.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = Examples
 
 Here you will find simple but complete examples of how to convert documents using the different configuration options available.
-The examples cover the main scenarios, but you can mix and match usage of instances, builders and maps freely to suite yous needs.
+The examples cover the main scenarios, but you can mix and match usage of instances, builders and maps freely to suite yours needs.
```
[the remark this answers is not in the document]
Comment (Mogztter) [reply docs/modules/ROOT/pages/conversion-examples.adoc:4]: Correct, forgot to remove the "s" 🤦‍♂️

[the remark this answers is not in the document]
Comment (Mogztter) [reply docs/modules/ROOT/pages/conversion-examples.adoc:4]:
```suggestion
The examples cover the main scenarios, but you can mix and match usage of instances, builders and maps freely to suite your needs.
```

**Comment by robertpanzer** [comment]
Thanks! I'll cherry pick this also into the v2.5.x branch

**Comment by robertpanzer** [comment]
> Thanks! I'll cherry pick this also into the v2.5.x branch

Stupid me, this is already for 2.5.x, I'll cherry pick it into the main branch.

**Pull request merged**